Sources/NIOTransportServices/NIOTSChannelOptions.swift

```diff
@@ -28,6 +28,8 @@ public struct NIOTSChannelOptions {
 
     public static let currentPath = NIOTSChannelOptions.Types.NIOTSCurrentPathOption()
 
+    public static let maxReceiveBytes = NIOTSChannelOptions.Types.NIOMaxReceiveBytesOption()
```

Comment:
@Lukasa I think a separate option (as suggested in this PR) is actually better than matching the socket channel with a RecvAllocator, wdyt? We can't sensibly control the strategy that NW.fw will use so I'm actually happy with this as is, wdyt?